cli/cmdcontext/cmdcontext_test.go

```diff
@@ -79,3 +79,21 @@ exit 1`),
 	assert.ErrorContains(t, err, "failed to get tarantool version: exit status 1")
 	assert.Equal(t, version.Version{}, tntVersion)
 }
+
+func TestTtCli_GetVersion(t *testing.T) {
+	tmpDir := t.TempDir()
+
+	err := os.WriteFile(filepath.Join(tmpDir, "tt.sh"),
+		[]byte(`#!/bin/bash
+echo "Tarantool CLI version 2.3.1, linux/amd64. commit: a38556f (v2.3.1-11-ga38556f)"`),
```

Comment:
I have such output with the tt-ee:
```
$ ./tt version
Tarantool CLI EE 2.2.1, linux/amd64. commit: b9f41d0
```
It doesn't look like this will always work.